node/src/GlideClusterClient.ts

```diff
@@ -504,6 +523,10 @@ export class GlideClusterClient extends BaseClient {
         }
 
         this.configurePubsub(options, configuration);
+        this.configureAdvancedConfigurationBase(
```

Comment:
same here, is the order of the params should be the same as with configurePubsub?